recipes/structureharvester/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+cp $SRC_DIR/*.py $PREFIX/bin && chmod +x $PREFIX/bin/structureHarvester.py
```

Comment:
Does this mean that other (non-executable) python files will pollute $PREFIX/bin? If so, please put them into $PREFIX/lib/structureharvester instead, and have a bash wrapper or symlink that points to the main python script.

Reply:
There is only on other file.py which indeed isn't executable.
But sorry, I'm not sure to understand this alternative:
`symlink that points to the main python script.`
Do you have an example in mind?